common/app/common/configuration.scala

```diff
@@ -360,7 +360,8 @@ class GuardianConfiguration(val application: String, val webappConfDirectory: St
         oauthCallback <- configuration.getStringProperty("faciatool.oauth.callback")
       } yield OAuthCredentials(oauthClientId, oauthSecret, oauthCallback)
 
-    val showTestContainers = configuration.getStringProperty("faciatool.show_test_containers").exists(_ == "true")
+    val showTestContainers =
+      configuration.getStringProperty("faciatool.show_test_containers").contains("true")
```

Comment:
`Option.contains` isn't in scala 2.10. It's bitten me a few times though.

Might want to double check what `PROD` is on, but I think this has failed in teamcity for me before.